src/olympia/reviewers/forms.py

```diff
@@ -299,7 +302,25 @@ def create_option(self, *args, **kwargs):
         return option
 
 
-class ReviewForm(forms.Form):
+class WithAttachmentMixin:
+    def get_invalid_attachment_file_type_message(self):
+        valid_extensions_string = '(%s)' % ', '.join(VALID_ATTACHMENT_EXTENSIONS)
+        return gettext(
+            'Unsupported file type, please upload an archive '
+            'file {extensions}.'.format(extensions=valid_extensions_string)
+        )
```

Comment:
I think you can include this directly within `clean_attachment_file` - unless you're going to need to call it elsewhere and/or it's difficult to test otherwise